src/js/pages/system/OverviewDetailTab.js

```diff
@@ -145,8 +145,12 @@ class OverviewDetailTab extends mixin(StoreMixin) {
           </ConfigurationMapValue>
         </ConfigurationMapRow>
         <MountService.Mount
-          type="OverviewDetailTab:AdditionalGeneralDetails"
-          limit={2}
+          type="OverviewDetailTab:AdditionalGeneralDetails:Licensing:NodeCapacity"
```

Comment:
We tend to name mounted private plugins very generically, without specifically mentioning the enterprise feature in the code. For example, see the naming of mounted components in [secrets private plugin](https://github.com/mesosphere/dcos-ui-plugins-private/blob/master/secrets/hooks.js#L61)

Reply:
I pushed a more generic rename to this branch 😄